bots/naughty/rhel-7-4/6960-storaged-regression-lvm-resize

```diff
@@ -0,0 +1,5 @@
+  File "/build/cockpit/bots/../test/verify/check-storage-resize", line *, in testResize
+*
+Error resizing logical volume: Command-line `lvresize "TEST"/"vol" -L 209715200b -r' exited with non-zero exit status 5:
+stdout: `Do you want to unmount "/run/media/admin/FSYS"? [Y|n] n
+*
```

Comment:
For reference, downstream bug: https://bugzilla.redhat.com/show_bug.cgi?id=1462660